### ISSUE TITLE
Fix the upgrade steps doc - Upgrade/refresh charm channel

### ISSUE THREAD

**Issue by pedrofragola** [charmed-kubernetes/kubernetes-docs#796]
The doc[0] uses the sequence below to upgrade from 1.26 to 1.27

```shell
juju upgrade-charm kubernetes-control-plane
juju config kubernetes-control-plane channel=1.25/stable
```

It's important to add the step to upgrade the charm channel

```shell
juju refresh kubernetes-control-plane --channel=1.27/stable
```

Also changing the channel to 1.27 instead 1.25

Note: it's necessary to add the step to upgrade all charms channel.

[0] https://ubuntu.com/kubernetes/docs/1.27/upgrading

**Comment by kwmonroe** [comment]
Thanks for the report and solution!

Fwiw, this stems from a time when charms didn't have named tracks and so `upgrade-charm` was good enough to ensure the latest charms were deployed.  Refreshing to a new named track is now the right way to go; this fix will help prevent bugs like [lp:2017353](https://bugs.launchpad.net/charm-kubernetes-master/+bug/2017353).  huzzah!

**Comment by addyess** [comment]
https://github.com/charmed-kubernetes/kubernetes-docs/pull/797
